Sources/EdgeNetworkHandlers/CompletionHandlersManager.swift

```diff
@@ -48,13 +52,20 @@ class CompletionHandlersManager {
     func unregisterCompletionHandler(forRequestEventId: String) {
         guard !forRequestEventId.isEmpty else { return }
 
-        if let completionHandler = completionHandlers[forRequestEventId] {
-            completionHandler(edgeEventHandles[forRequestEventId] ?? [])
-            _ = completionHandlers.removeValue(forKey: forRequestEventId)
-            Log.trace(label: TAG, "Removing completion handler for Edge response with request event id \(forRequestEventId).")
+        queue.async { [weak self] in
+
+            guard let self = self else { return }
+
+            if let completionHandler = completionHandlers[forRequestEventId] {
+                completionHandler(edgeEventHandles[forRequestEventId] ?? [])
+
+                Log.trace(label: TAG, "Removing completion handler for Edge response with request event id \(forRequestEventId).")
+                completionHandlers.removeValue(forKey: forRequestEventId)
```

Comment:
This my produce a lint warning since the response is ignored. Do you need to include `_ =` to the call?